deploying-a-django-app-on-okteto.md

```diff
@@ -0,0 +1,704 @@
+# Deploying a Django App on Okteto
+
+## Introduction
+
+In this tutorial, We will building and deploying a Django Role based Authorization Service API(RBAS) on Okteto.
+
+An RBAS application, is a system with checks to give access to users based on their roles in the system. 
```

Comment:
```suggestion
An RBAC application, is a system with checks to give access to users based on their roles in the system. 
```

Reply:
could you add a sentence or two listing why this type of applications are important, or giving an example of what using RBAC allows?